src/include/catalog/pg_proc.dat

```diff
@@ -12433,6 +12433,11 @@
   proargtypes => 'regclass name bool',
   proargnames => '{relation,attname,inherited}',
   prosrc => 'pg_clear_attribute_stats' },
+{ oid => '9543',
+  descr => 'statistics: commit time, in milliseconds',
+  proname => 'pg_stat_get_db_commit_time', provolatile => 's',
+  proparallel => 'r', prorettype => 'float8', proargtypes => 'oid',
+  prosrc => 'pg_stat_get_db_commit_time' },
```

Comment:
You should keep this grouped with the other pg_stat_get functions, just after the last pg_stat_get_db_parallel_workers_launched